vars/AlmaLinux_10.yml

```diff
@@ -0,0 +1,14 @@
+---
+blivet_package_list:
+  - python3-blivet
+  - libblockdev-crypto
+  - libblockdev-dm
+  - libblockdev-lvm
+  - libblockdev-mdraid
+  - libblockdev-swap
+  - xfsprogs
+  - stratisd
+  - stratis-cli
+# vdo not yet available on el10
+#  - vdo
+#  - kmod-kvdo
```

Comment:
This will be just the `vdo` package for 10, the kernel module will be builtin in the kernel (and called dm-vdo).

Reply:
Yes. I think something like `modprobe --dry-run kvdo || modprobe --dry-run dm-vdo` should work.

Reply:
ok - once we have vdo support in el10, I will submit another PR to enable tests